web/angular-wallet/src/app/main/dashboard/dashboard.component.html

```diff
@@ -5,7 +5,8 @@
     <div class="mr-lg-32" fxFlex="100" *ngIf="account">
       <mat-toolbar-row *ngIf="!account.confirmed && settings.welcomeMessageHiddenFrom.indexOf(account.account) === -1">
         <mat-icon>card_giftcard</mat-icon>
-        <span>{{ 'welcome_to_burst' | i18n }} <a href="http://faucet.burst-alliance.org:8080" target="_blank">{{ 'click_to_claim' | i18n }}</a></span>
+        <span>{{ 'welcome_to_burst' | i18n }} <a href="http://faucet.burst-alliance.org:8080"
```

Comment:
fyi I think this URL is changing to lose the port 8080 and run on https. We may want to update this before launch if Jello gets it done.

Reply:
should be configurable

Reply:
changed to https://faucet.burst-alliance.org